lib/app/router/main_tabs/main_tab_navigation.dart

```diff
@@ -93,25 +93,24 @@ class _BottomNavBarContent extends StatelessWidget {
                     ),
                   ],
           ),
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: TabItem.values.map((tabItem) {
-              final isSelected = currentTab == tabItem;
-              return Expanded(
-                child: GestureDetector(
-                  behavior: HitTestBehavior.opaque,
-                  onTap: () => onTabPressed(tabItem),
-                  child: Container(
-                    padding: EdgeInsets.only(
-                      top: 9.0.s,
-                      bottom: MediaQuery.of(context).padding.bottom > 0 ? 23.0.s : 9.0.s,
+          child: SafeArea(
```

Comment:
i think it ll be good to sync with designer to do not broke exist implementation of bottom navigation bar.